src/main/resources/theme/chat/chat.fxml

```diff
@@ -14,7 +14,7 @@
     <children>
         <TabPane fx:id="tabPane" tabClosingPolicy="ALL_TABS" AnchorPane.bottomAnchor="0.0" AnchorPane.leftAnchor="0.0"
                  AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0" visible="false">
-            <Tab text="+" closable="false">
+            <Tab fx:id="addChannelTab" text="+" closable="false">
```

Comment:
The fx:id value actually gets set as the id for the node. So you shouldn't need to set the id in the controller itself.

Reply:
Oh! I'll know, thanks. But we don't need the id anymore. Found a more elegant solution using `isClosable` condition